packages/checkbox/src/react/__specs__/__snapshots__/storyshots.spec.ts.snap

```diff
@@ -279,10 +270,9 @@ exports[`Storyshots Checkbox error 1`] = `
       <input
         checked={true}
         data-css-gyrv08=""
-        disabled={false}
```

Comment:
why remove?

Reply:
This is apparently just a difference with how the `...filterReactProps` and `...props` behavior works.

disabled=false === enabled, so it's not really required anyway